typescript error after new release (4.5.1)

4.5.1 causes the following errors
```

../../node_modules/keyv/src/index.d.ts(92,3): error TS1070: 'async' modifier cannot appear on a type member.
../../node_modules/keyv/src/index.d.ts(93,3): error TS1070: 'async' modifier cannot appear on a type member.
../../node_modules/keyv/src/index.d.ts(94,3): error TS1070: 'async' modifier cannot appear on a type member.
../../node_modules/keyv/src/index.d.ts(95,3): error TS1070: 'async' modifier cannot appear on a type member.

```

Typescript version is 4.8.4
Node version is 16.0.0

I'm getting the same on my builds just now:

```shell

node_modules/keyv/src/index.d.ts:92:3 - error TS1070: 'async' modifier cannot appear on a type member.

92      async compress(value: any, options?: any);
        ~~~~~

node_modules/keyv/src/index.d.ts:92:3 - error TS7010: 'compress', which lacks return-type annotation, implicitly has an 'any' return type.

92      async compress(value: any, options?: any);
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/keyv/src/index.d.ts:93:3 - error TS1070: 'async' modifier cannot appear on a type member.

93      async decompress(value: any, options?: any);
        ~~~~~

node_modules/keyv/src/index.d.ts:93:3 - error TS7010: 'decompress', which lacks return-type annotation, implicitly has an 'any' return type.

93      async decompress(value: any, options?: any);
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/keyv/src/index.d.ts:94:3 - error TS1070: 'async' modifier cannot appear on a type member.

94      async serialize(value: any);
        ~~~~~

node_modules/keyv/src/index.d.ts:94:3 - error TS7010: 'serialize', which lacks return-type annotation, implicitly has an 'any' return type.

94      async serialize(value: any);
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/keyv/src/index.d.ts:95:3 - error TS1070: 'async' modifier cannot appear on a type member.

95      async deserialize(value: any);
        ~~~~~

node_modules/keyv/src/index.d.ts:95:3 - error TS7010: 'deserialize', which lacks return-type annotation, implicitly has an 'any' return type.

95      async deserialize(value: any);
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 8 errors.
```

--- a/packages/keyv/src/index.d.ts
+++ b/packages/keyv/src/index.d.ts
@@ -89,10 +89,10 @@ declare namespace Keyv {
 	}
 
 	interface CompressionAdapter {
-		async compress(value: any, options?: any);
-		async decompress(value: any, options?: any);
-		async serialize(value: any);
-		async deserialize(value: any);
+		compress(value: any, options?: any): Promise<any>;
+		decompress(value: any, options?: any): Promise<any>;
+		serialize(value: any): Promise<any>;
+		deserialize(value: any): Promise<any>;
 	}
 
 	interface DeserializedData<Value> {

